nixops/resources/output.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+
+# Arbitrary JSON.
+
+import os
+import nixops.util
+import nixops.resources
+
+import tempfile
+import shutil
+import subprocess
+import hashlib
+from typing import Any, Optional, List, Dict, AnyStr, Tuple
+
+class OutputDefinition(nixops.resources.ResourceDefinition):
+    """Definition of an Output."""
+
+    @classmethod
+    def get_type(cls):
+        # type: () -> (str)
+        return "output"
+
+    @classmethod
+    def get_resource_type(cls):
+        # type: () -> (str)
+        return "output"
+
+    def __init__(self, xml):
+        # type: (Any) -> None
```

Comment:
This should be `xml.etree.elementtree.Element`.

Reply:
fixed